cli/fmt_errors.rs

```diff
@@ -10,6 +10,15 @@ use std::ops::Deref;
 
 const SOURCE_ABBREV_THRESHOLD: usize = 150;
 
+pub fn format_location(filename: String, line: i64, col: i64) -> String {
+  format!(
+    "{}:{}:{}",
+    colors::cyan(filename),
+    colors::yellow(line.to_string()),
+    colors::yellow(col.to_string())
+  )
+}
```

Comment:
It’s unfortunate that we have two implementations of format_location.. but I guess the other is in JS